services/registration/src/models/branch.ts

```diff
@@ -20,17 +21,43 @@ export interface Branch {
 }
 
 const branchSchema = new Schema<Branch>({
-  name: { type: String, required: true },
-  type: { type: String, required: true, enum: BranchType },
+  name: {
+    type: String,
+    required: true,
+  },
+  hexathon: {
+    type: Number,
+    required: true,
+  },
+  type: {
+    type: String,
+    required: true,
+    enum: BranchType,
+  },
   settings: {
-    open: { type: Date, required: true },
-    close: { type: Date, required: true },
+    open: {
+      type: Date,
+      required: true,
+    },
+    close: {
+      type: Date,
+      required: true,
+    },
   },
   formPages: [
     {
-      title: { type: String, required: true },
-      jsonSchema: { type: String, required: true },
-      uiSchema: { type: String, required: true },
+      title: {
+        type: String,
+        required: true,
+      },
+      jsonSchema: {
+        type: Object,
+        required: true,
+      },
+      uiSchema: {
+        type: Object,
+        required: true,
```

Comment:
These should be kept as type `string`, I don't think this change is needed unless there is a reason?